components/Snackbar/src/MDCSnackbarMessageView.m

```diff
@@ -725,9 +739,25 @@ - (CGSize)intrinsicContentSize {
   // Make sure that the height of the image and text is larger than the minimum height;
   height = MAX(kMinimumHeight, height);
 
+  height = MAX(kMinimumHeight, height) + [self contentSafeBottomInset];
   return CGSizeMake(UIViewNoIntrinsicMetric, height);
 }
 
+- (CGFloat)contentSafeBottomInset {
+  // If a bottom offset has been set to raise the HUD, e.g. above a tab bar, we should ignore
+  // any safeAreaInsets, since it is no longer 'anchored' to the bottom of the screen. This is set
+  // by the GOOHUDOverlayView whenever the bottomOffset is non-zero.
+  if (!self.anchoredToScreenEdge) {
+    return 0;
+  }
+#if defined(__IPHONE_11_0) && (__IPHONE_OS_VERSION_MAX_ALLOWED >= __IPHONE_11_0)
+  if (@available(iOS 11.0, *)) {
+    return self.window.safeAreaInsets.bottom;
```

Comment:
Since this uses the safeAreaInsets, do we need to update the constraint that is using it when the safe area changes? Or do we nuke the view(s) on rotation?

Reply:
They get updated through the MDCSnackbarOverlayView forcing a relayout... Not ideal but I think this is the simplest solution.

Reply:
Oh I see, sgtm, just make sure this also works for the horizontal insets you just changed.